src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -70,8 +116,13 @@ abstract class LazyModule()(implicit val p: Parameters)
     buf.toString
   }
 
+  /** give index to this instance. */
```

Comment:
what is an index? Is the index the index of this module in it's parent's children list? Or a global list of all lazy modules...?

Reply:
changed to 
```
  /** give global [[LazyModule]] index to this instance. */
```